front/src/components/Messages.js

```diff
@@ -55,14 +55,21 @@ const MessagesDiv = styled.div`
 
 
 const Messages = (props) => {
+  const [isNavigationMode, setNavigationMode] = useState()
```

Comment:
`const [isNavigationMode, setIsNavigationMode] = useState(false)`
is better

Reply:
Can I push another commit for this edit or should I amend by last commit? How do you prefer things?

Reply:
push another. It's best to avoid changing public history.

Reply:
Cool. This is done.